src/event-stream/event-server.ts

```diff
@@ -495,6 +585,63 @@ export async function startEventServer(opts: {
     }
   });
 
+  app.postAsync('/attachments/new', async (req, res) => {
+    const body = req.body;
+    for (const attachment of body) {
+      const metadataCV: TupleCV = deserializeCV(hexToBuffer(attachment.metadata)) as TupleCV;
```

Comment:
before anything, when receiving an attachment, can you make sure that the `attachment.contract_id` is BNS?